osf/models/registrations.py

```diff
@@ -1126,7 +1126,7 @@ def create_from_node(cls, user, schema, node=None, data=None, provider=None):
             provider=provider,
         )
         draft.save()
-        draft.copy_editable_fields(node, Auth(user), save=True, contributors=False)
+        draft.copy_editable_fields(node, Auth(user), save=True, contributors=True)
```

Comment:
Could you remove the contributors argument from copy_editable_fields and the related if branch? i added this to fix a contributors related bug. Now that contributors are independent, we don't need that if conditional anymore!